recipes/dask-searchcv/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dask-searchcv" %}
+{% set version = "0.0.1" %}
+{% set md5 = "7d8fb38ba01d7518e2d22e5933f1e419" %}
```

Comment:
Not sure, it's probably just a strong advise (seeing how it's systematically mentionned by core developpers in reviews e.g. https://github.com/conda-forge/staged-recipes/pull/2246#discussion_r105520584)  Also pypi.org uses sha256 https://pypi.org/project/dask-searchcv/#files